assets/js/blocks/cart-checkout/mini-cart/index.tsx

```diff
@@ -26,6 +26,9 @@ const settings = {
 	supports: {
 		html: false,
 		multiple: false,
+		color: true,
+		__experimentalSelector:
+			'.wc-block-mini-cart__button, .wc-block-mini-cart__badge',
```

Comment:
If you remove this, the global style doesn't work correctly. Without this, Gutenberg will add global style to `.wp-block-woocommerce-mini-cart` instead of the above selectors. This is fine for text color (by inheriting the parent element) but doesn't work for background color.

Reply:
Is the GB team aware that we're using this experimental supports property and why we need to use it? If not, it might be handy to make sure they know somehow so there's awareness of it's usefulness for other projects when it comes time to evaluate graduating it.

Reply:
@nerrad I don't think they know about it. I will add comments explaining why we need them.